frontend/app/actions/Fallbacks.scala

```diff
@@ -18,14 +18,10 @@ object Fallbacks {
   def memberHome(implicit request: RequestHeader) =
     redirectTo(controllers.routes.FrontPage.welcome)
 
-  def notYetAMemberOn(implicit request: RequestHeader) =
-    redirectTo(controllers.routes.Joiner.tierChooser()).addingToSession("preJoinReturnUrl" -> request.uri)
+  def notYetAMemberOn(implicit request: RequestHeader) = Forbidden
```

Comment:
could use Gone or NotFound
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/410

Reply:
I think the thing is that it does still exist for members so it's not gone, but if you are not a member it is forbidden